### DISCUSSION TITLE
Does ccache support Microsoft Visual C++ 2017?

### DISCUSSION THREAD

**Discussion by dyadav7** [ccache/ccache#682] [Questions & answers] [answered]
### Question ###

Does ccahe support Microsoft Visual C++ 2017 ?

**Comment by AlexanderLanin** [comment, marked as answer]
Short Answer: no
Details here: https://ccache.dev/platform-compiler-language-support.html

**Comment by dyadav7** [comment]
I will wait for Visual C++ 2017 support.
